content/_help/verify-your-identity/overview._en.md

```diff
@@ -6,23 +6,68 @@ permalink: /help/verify-your-identity/overview/
 meta_title: Overview
 order: 0
 redirect_from:
-  - /help/verify-your-identity/
-  - /en/help/verify-your-identity/
-  - /en/help/verify-your-identity/overview/
-  - /help/identity-verification/i-have-more-than-one-logingov-account-can-I-verify-my-identity-for-all-of-them/
-  - /help/identity-verification/related-articles/
-  - /help/identity-verification/why-do-i-need-to-verify-my-identity/
-  - /help/verifying-your-identity/multiple-accounts-to-verifying-my-identity-for/
-  - /help/verifying-your-identity/why-do-i-need-to-verify-my-identity/
+- /help/verify-your-identity/
+- /en/help/verify-your-identity/
+- /en/help/verify-your-identity/overview/
+- /help/identity-verification/i-have-more-than-one-logingov-account-can-I-verify-my-identity-for-all-of-them/
+- /help/identity-verification/related-articles/
+- /help/identity-verification/why-do-i-need-to-verify-my-identity/
+- /help/verifying-your-identity/multiple-accounts-to-verifying-my-identity-for/
+- /help/verifying-your-identity/why-do-i-need-to-verify-my-identity/
 ---
 
-Some participating agencies require you to complete an identity verification process. Identity verification is the process where you prove you are you - and not someone pretending to be you.
+Identity verification is the process where you prove you are you — and not someone pretending to be you. You will need to [create your Login.gov account](/create-an-account) and verify your identity to access services at some government agencies.
 
-You only need to verify your identity once for your Login.gov account. After you verify your identity with Login.gov for one government application, you don’t need to do it again for other government applications that use Login.gov and require identity verification. You currently do not have to upload a picture of yourself or take a selfie for identity verification. However, you may have to take a picture of your ID.
+## What you’ll need to verify your identity
 
-Use this help section to understand the process and requirements for identity verification.
+You’ll take a photo of your [accepted driver’s license or state ID card](/help/verify-your-identity/accepted-identification-documents) and enter your personal information to verify your identity online.
 
-## Common troubleshooting topics
+You may need to take a [photo of yourself](/help/verify-your-identity/how-to-add-images-of-your-state-issued-id) with a phone to verify your identity online. This is to check that you are the person on your ID. If you don’t have a phone with a camera, you can [verify your identity in person](/help/verify-your-identity/verify-your-identity-in-person).
 
-{: .help-question-list}
-* [How to verify your identity](/help/verify-your-identity/how-to-verify-your-identity/)
+<div class="grid-row grid-gap measure-3">
+
+  <div class="tablet:grid-col">
```

Comment:
do you think it would be worth trying to create a partial for this two-column icon setup?

something that could be like...

```liquid
{%
  partial two_image_bullet_list.html
  image1="/assets/img/help/create-an-account.png" 
  text1="What you’ll need to create a Login.gov account"
  bullets1=["Email","Password","Authentication method"]
  image2="/assets/img/help/verify-your-identity.png"
  text2="What you’ll need to verify your identity"
  bullets2=["Driver’s license or state ID card", "Social Security number", "Phone number or address"]
  after_text2="Some services need you to take a photo of yourself with a phone to verify your identity online."
%}
```

Reply:
Extracted a couple of partials there, actually, see what you think of the result.